src/traintest.jl

```diff
@@ -1,10 +1,13 @@
 
 function train(dre::DRE, eeg_in, eeg_out, ps, st; n_epochs=1, lr=0.01, batch_size=32, show_progress=true)
-    @info "Input data is a $(typeof(eeg_in)) and $(typeof(eeg_out)) with size $(size(eeg_in)),$(size(eeg_out))"
 
+    @info "Input data is a $(typeof(eeg_in)) and $(typeof(eeg_out)) with size $(size(eeg_in)),$(size(eeg_out))"
     # conv layer expects input of shape (time, channel, epoch)
     opt_state = create_optimiser(ps, lr)
     p = Progress(n_epochs)
+    
+    loss_epoch_array = Float64[]
+    loss_epoch_array = Array{Float64}(undef,n_epochs)
```

Comment:
you do not need the first line here, as it is immediately overwritten by the second one